### PR TITLE
Allow public IP logging

### PR DESCRIPTION
**Pull request by TheNavigat**

### CODE REVIEW

**Comment by TheNavigat** [comment]
The test failure is really confusing for me...

I tested the script locally, and the exception is raised correctly if ```--manual-allow-public-ip-logging``` is not passed. However, the testing script reports that the script continue running even after disagreeing with logging the public IP. I can't seem to be able to understand why does that happen.

**Comment by TheNavigat** [comment]
I added a couple of print statements before the ```if not self.conf("public-ip-logging-ok")``` condition, turns out that for some reason MagicMock sets the argument to some value.

```
Public IP Logging:
<MagicMock name='mock.manual_public_ip_logging_ok' id='4407338960'>
```

File: letsencrypt/plugins/manual.py [hunk 1]
```diff
@@ -90,6 +90,8 @@ def __init__(self, *args, **kwargs):
     def add_parser_arguments(cls, add):
         add("test-mode", action="store_true",
             help="Test mode. Executes the manual command in subprocess.")
+        add("public-ip-logging-ok", action="store_true",
```
Comment (bmw) [inline letsencrypt/plugins/manual.py]: Small suggestion: This becomes the flag `--manual-public-ip-logging-ok`. This is very verbose. What about `--manual-ip-logging-ok` or `--manual-public-log-ok`?

**Comment by bmw** [comment]
Other than my very small suggestion, this LGTM.

**Comment by TheNavigat** [comment]
As the whole PR is a solution to @kuba's issue, that's where I got the name from.

I'm siding with the more verbose one. A long flag is better than a short but an ambiguous one.

**Comment by bmw** [comment]
Very true. After thinking about it, I think the flag is fine.

LGTM. Thanks @TheNavigat.

**Pull request merged**